### DISCUSSION TITLE
modify esmerald ValidationErrorException in msgspec

### DISCUSSION THREAD

**Discussion by saurabhgharatpara** [dymmond/esmerald#385] [Q&A] [answered]
![validationerror](https://github.com/user-attachments/assets/83eee6b5-79f6-419d-942e-239f04798fc5)
Whenever msgspec exception happens esmerald.exceptions.ValidationErrorException is raised along with response for e.g 
{
  "message": "Validation failed for http://127.0.0.1:8000/admin/pets  with method POST.",
  "extra": {
    "extra": [
      {
        "name": "Expected `str`, got `int`"
      }
    ]
  }
}
basically i need  response in this format 
{
  "message": "Validation failed for http://127.0.0.1:8000/admin/lob with method POST.",

}

what can we do ?

**Comment by tarsil** [comment]
@saurabhgharatpara thank you for the message. Lets us analyse this and see how we can help you.

**Comment by tarsil** [comment]
@saurabhgharatpara have you tried to use the newest https://esmerald.dymmond.com/exceptions/#validatorerror

?

**Reply by saurabhgharatpara** [reply, marked as answer]
Yes, im implementing by modifying the custom exception
